known-issues/Issues.txt

```diff
@@ -0,0 +1,12 @@
+1. update Readme.md
+
+2. 3rd party modules in general:
```

Comment:
I think this point is the main problem with the implementation of the UI over Docker. If someone is trying to implement a 3rd party module, which needs another dependency, the user is required to modify the Dockerfile and build it by himself. Or we get a ton of "Add this or that dependency" as issues.

What do you think about a little section in the README which describes the way on how to use the UI with Docker. We could create a subfolder with an example Dockerfile.